modules/common/src/main/java/org/opencastproject/mediapackage/AdaptivePlaylist.java

```diff
@@ -82,15 +81,12 @@
   // Known tags that references other files include the following - but we only use EXT-X-MAP here
   // "#EXT-X-MAP:", "#EXT-X-MEDIA:", "#EXT-X-I-FRAME-STREAM-INF:", "#EXT-X-SESSION-DATA:",
   // Variant tags: see Section 4.4.2 in draft
-  List<String> extVariant = new ArrayList<String>(
-          Arrays.asList("#EXT-X-MAP:", "#EXT-X-TARGETDURATION:", "EXTINF", "#EXT-X-BYTERANGE:"));
+  List<String> extVariant = Arrays.asList("#EXT-X-MAP:", "#EXT-X-TARGETDURATION:", "EXTINF", "#EXT-X-BYTERANGE:");
   // Master tags: see Section 4.4.4
-  List<String> extMaster = new ArrayList<String>(
-          Arrays.asList("#EXT-X-MEDIA:", "#EXT-X-STREAM-INF:", "#EXT-X-I-FRAME-STREAM-INF:", "#EXT-X-SESSION-DATA:"));
-  Pattern masterPatt = Pattern.compile(extMaster.stream().collect(Collectors.joining("|")), Pattern.CASE_INSENSITIVE);
-  Pattern variantPatt = Pattern.compile(extVariant.stream().collect(Collectors.joining("|")), Pattern.CASE_INSENSITIVE);
-  Predicate<String> masterTags = f -> masterPatt.matcher(f) != null;
-  Predicate<String> variantTags = f -> variantPatt.matcher(f) != null;
+  List<String> extMaster = Arrays.asList("#EXT-X-MEDIA:", "#EXT-X-STREAM-INF:", "#EXT-X-I-FRAME-STREAM-INF:",
+          "#EXT-X-SESSION-DATA:");
+  Pattern masterPatt = Pattern.compile(String.join("|", extMaster), Pattern.CASE_INSENSITIVE);
+  Pattern variantPatt = Pattern.compile(String.join("|", extVariant), Pattern.CASE_INSENSITIVE);
```

Comment:
Mostly simplifications, but `masterPatt` and `variantPatt` always evaluate to `true` and make no sense. Luckily, they are also not used.